torch/csrc/jit/custom_operator.h

```diff
@@ -170,7 +170,8 @@ FunctionSchema inferAndCheckSchema(const std::string& schemaOrName) {
 template <typename Implementation>
 Operator createOperator(
     const std::string& schemaOrName,
-    Implementation&& implementation) {
+    Implementation&& implementation,
+    OperatorOptions options = OperatorOptions()) {
```

Comment:
These should be exposed in `RegisterOperators::op()` as well since that's the suggested way to register ops in the docs/tutorials